web-compile-config.yml

```
@@ -1,7 +1,7 @@
 web-compile:
   sass:
     files:
-      src/scss/sphinx-book-theme.scss: sphinx_book_theme/static/sphinx-book-theme.[hash].css
+      src/scss/sphinx-book-theme.scss: sphinx_book_theme/static/sphinx-book-theme.css
```

I'm guessing you looked at this, but is there a reason that this wouldn't work?

```suggestion
      src/scss/sphinx-book-theme.scss: sphinx_book_theme/static/sphinx-book-theme.css?digest=[hash]
```

---

This is the name of the file to generate from the sass file.

---

yeah but the previous version of it was:

```
src/scss/sphinx-book-theme.scss: sphinx_book_theme/static/sphinx-book-theme.[hash].css
```

so if we just move the `[hash]` to the end wouldn't it get populated as expected?

```
src/scss/sphinx-book-theme.scss: sphinx_book_theme/static/sphinx-book-theme.css?digest=[hash]
```

Maybe I am not understanding something. :-/

---

ahh I see we don't actually want the hash in the filename, disregard this comment

---

Oh yeh wait this is not correct, it would be `src/scss/sphinx-book-theme.scss: sphinx_book_theme/static/sphinx-book-theme.css`, but then in the actual template is where the digest needs to be added. Is this possible in the sphinx machinery?